.gitlab-ci.yml

```diff
@@ -39,6 +39,7 @@ smoke-tests:
   - .gitlab/pipelines/stages/deploy/deploy_kubecf.sh
   - .gitlab/pipelines/stages/deploy/wait_kubecf.sh
   - .gitlab/pipelines/stages/tests/smoke_tests.sh
+  - .gitlab/pipelines/stages/deploy/stop_k8s.sh
```

Comment:
Shouldn't this be an `after_script`?

Reply:
No, because it's running in the same container.